face/faces/WebFace/BaseWebFace.js

```diff
@@ -170,8 +171,8 @@ export default class BaseWebFace extends BaseFace {
       if (vatom.id === this.vatom.id && this.face) {
         var resources = {}
 
-        for (var res in this.vatomView.vatom.resources) {
-          resources[res] = this.vatomView.vatom.resources[res].value.value
+        for (var res in this.vatomView.vatom.properties.resources) {
```

Comment:
This is using `in` on an array, so it would end up with an object with keys 0, 1, 2 etc... Maybe something like this rather:

``` js
var resources = {}
for (var resource of this.vatomView.vatom.properties.resources)
  resources[resource.name] = resource.value.value
```

Reply:
d706009